File: https:/github.com/abhinavtripathi17/DSA-Leetcode.git/628-MaximumProductOfThreeNumbers/628-MaximumProductOfThreeNumbers.cpp
// Last updated: 8/4/2026, 7:14:19 PM
class Solution {
public:
    int maximumProduct(vector<int>& nums) {
        int n = nums.size();
        sort(nums.begin() , nums.end());
        return max(nums[0]* nums[1]*nums[n-1] , nums[n-1]*nums[n-2]*nums[n-3]);

    }
};

File: https:/github.com/abhinavtripathi17/DSA-Leetcode.git/628-MaximumProductOfThreeNumbers/628-MaximumProductOfThreeNumbers.java
// Last updated: 8/4/2026, 7:14:25 PM
class Solution {
    public int maximumProduct(int[] nums) {
        int n = nums.length;
        Arrays.sort(nums);
        int maxi1 = nums[n-1] * nums[n-2] * nums[n-3];
        int maxi2 = nums[0] * nums[1] * nums[n-1];

        return Math.max(maxi1 , maxi2);
        
        
        
         
    }
}